db/schema.rb

```diff
@@ -72,6 +72,10 @@
     t.string "unconfirmed_email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "full_name"
+    t.string "phone"
+    t.float "rate_per_hour"
```

Comment:
I don't believe they work on rate system. They have flat rate per day + commission from extra sales. This is premature implementation.

Reply:
we can set rate (flat rate per day) / hours